app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError. "
```

Comment:
Remove text like NotVaccinatedError. we will already know it due to raise type error